src/bci_build/package/__init__.py

```diff
@@ -1226,7 +1235,16 @@ def _tag_variant(self) -> str:
 
     @property
     def uid(self) -> str:
-        return f"{self.name}-{self._tag_variant}" if self.version_in_uid else self.name
+        registry = (
+            "-sac"
+            if isinstance(self._publish_registry, ApplicationCollectionRegistry)
+            else ""
+        )
+        return (
+            f"{self.name}-{self._tag_variant}{registry}"
+            if self.version_in_uid
+            else self.name
+        )
```

Comment:
Please don't do this, it'll break the crb yaml PR

Reply:
What should I be doing instead? I need to generate two python 3.11 containers, one for registry and one for SAC. I don't see a way other than doing a special version of the uid. 

Why does this break the crb generator and why can this not be fixed? the yaml generator shouldn't care about the uid (or if it generates this into the yaml key, then we need to adjust the keys accordingly)?

Reply:
would a prefix be helping instead of a suffix?